.green

```diff
@@ -0,0 +1,4 @@
+verbose       = 2
+omit-patterns = tortoise/contrib/pylint/*,tortoise/tests/*
+clear-omit    = true
+run-coverage  = true
```

Comment:
Tests seems to fall because of concurrently trying to create many databases with same names and concurrently trying to destroying it by everybody.
It could be avoided by making `processes = 1` in this config, but some problems will still persist.
You should add some unique token to db name so each test works on it's own database - or you could make some kind of global creating database and then running tests in transactions, something like I am doing in my [web template](https://github.com/Zeliboba5/aiohttp-tortoise-boilerplate/blob/master/conftest.py). But I think for tortoise is better to make db for each test independently, just need to think of some good way to clean them up after failed tests

Reply:
No, each db it creates has a randomised name, So it should work fine.
It is an extrapolation of the `{}` in  `TORTOISE_TEST_DB=postgres://postgres:@127.0.0.1:5432/test_{}`.

I was considering making `Tortoise.init()` take a db_url but for testing it was simpler to use `await self.getDB()` which would create a new db that will auto-tear down. (which is subject to change)